index.html

```diff
@@ -767,6 +759,9 @@ <h2 id="3Definitions">Definitions</h2>
     will display this rather than the animation.
   </dd>
 
+<dt><dfn>transfer function</dfn></dt>
+<dd>function relating the desired display output with the image samples</dd>
```

Comment:
This is true for display-referrred color spaces, but not for scene-referred color spaces.

Reply:
What about _function relating image luminance with image samples_

Reply:
See https://github.com/w3c/PNG-spec/pull/209/commits/b3c296d4b98baa4b0f2c74ee18cd5c71338f612e